app/uk/gov/hmrc/accessibilitystatementfrontend/config/SourceConfig.scala

```diff
@@ -17,42 +17,21 @@
 package uk.gov.hmrc.accessibilitystatementfrontend.config
 
 import javax.inject.{Inject, Singleton}
-import play.api.Logging
 import scala.io.Source
 
 case class StatementSource(source: Source, filename: String)
 
-trait SourceConfig extends Logging {
-  def statementsSource(): StatementSource
-
-  def statementSource(service: String): StatementSource
-}
-
-trait ProductionSourceConfig extends SourceConfig
-trait TestOnlySourceConfig extends SourceConfig
-
 @Singleton
-case class DefaultProductionSourceConfig @Inject()() extends ProductionSourceConfig {
-  override def statementsSource(): StatementSource = {
-    val filename = "services.yml"
-    StatementSource(Source.fromResource(filename), filename)
-  }
+case class SourceConfig @Inject()(appConfig: AppConfig) {
+  import appConfig._
 
-  override def statementSource(service: String): StatementSource = {
-    val filename = s"services/$service.yml"
-    StatementSource(Source.fromResource(filename), filename)
-  }
-}
-
-@Singleton
-case class DefaultTestOnlySourceConfig @Inject()() extends TestOnlySourceConfig {
-  override def statementsSource(): StatementSource = {
-    val filename = "testOnlyServices.yml"
+  def statementsSource(): StatementSource = {
```

Comment:
Just having a look at this, and `ServicesYamlFinder`. Should we create a ticket to delete the `ServicesYamlFinder` as a technical debt once we are happy that the classpath mechanism is working as expected in all the environments?

Reply:
In the meantime, could we have a comment on this class explaining why it exists in the codebase but isn't currently bound as the implementation of retrieving the list of statements.